client/me/billing-history/billing-history-table.jsx

```diff
@@ -46,17 +46,8 @@ module.exports = React.createClass( {
 				{ ...this.props }
 				initialFilter={ { date: { newest: 5 } } }
 				header
-				description={ function( transaction ) {
-					return (
-						<div className="transaction-links">
-							<a className="view-receipt" href={ '/me/billing/' + transaction.id } onClick={ this.recordClickEvent( 'View Receipt in Billing History' ) } >
-								{ this.translate( 'View Receipt' ) }
-							</a>
-							{ this.renderEmailAction( transaction.id ) }
-						</div>
-					);
-				}.bind( this ) }
-				/>
+				renderEmpty={ this.renderEmpty }
```

Comment:
How about merging this inside the `render()` function? I'm a bit leery of calling something `renderX()` when it doesn't produce React nodes/elements.

```js
render() {
	const emptyString = this.translate( … );

	…
	renderEmpty={ emptyString }
}
```